share/build/buildlib.ekat

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+import os, sys, argparse, logging, shutil
+
+from standard_script_setup import *
+from CIME import utils
+from CIME.utils import expect, run_bld_cmd_ensure_logging, run_cmd_no_fail, run_cmd
+from CIME.case import Case
+from CIME.build import get_standard_makefile_args
+
+sys.path.append(os.path.dirname(__file__))
+from buildlib_util import extract_from_macros
+
+logger = logging.getLogger(__name__)
+
+###############################################################################
+def parse_command_line(args, description):
+###############################################################################
+    parser = argparse.ArgumentParser(
+        usage="""\n{0} [--debug]
+OR
+{0} --verbose
+OR
+{0} --help
+
+\033[1mEXAMPLES:\033[0m
+    \033[1;32m# Run \033[0m
+    > {0}
+""".format(
+            os.path.basename(args[0])
+        ),
+        description=description,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    utils.setup_standard_logging_options(parser)
+
+    parser.add_argument("buildroot", help="build path root")
+
+    parser.add_argument("installpath", help="install path ")
+
+    parser.add_argument(
+        "caseroot", nargs="?", default=os.getcwd(), help="Case directory to build"
+    )
+
+    args = utils.parse_args_and_handle_standard_logging_options(args, parser)
+
+    return args.buildroot, args.installpath, args.caseroot
+
+###############################################################################
+def buildlib(bldroot, installpath, case):
+###############################################################################
+
+    # Figure out where we are getting Kokkos from
+    kokkos_dir = None
+    if case.get_value("USE_ALBANY") or case.get_value("USE_TRILINOS"):
+        print ("case is using Trilinos and will use its Kokkos")
+
+        # Set Kokkos_ROOT so Ekat knows to find Kokkos from Albany/Trilinos
+        expect(os.environ.get("Trilinos_ROOT") is not None,
+               "USE_TRILINOS|USE_ALBANY is On, but Trilinos_ROOT is not set!")
+        os.environ["Kokkos_ROOT"] = os.environ.get("Trilinos_ROOT")
+
+    elif os.environ.get("Kokkos_ROOT") is not None:
```

Comment:
Why do we care about this case? And why do we care about setting kokkos_dir?

Reply:
Ah, in buildlib.kokkos, `kokkos_dir` was used to get nvcc_wrapper, but I don't think we need that here, so we can just remove this stuff.